Update tables to include consensus cell types in report (#939)
AlexsLemonade/scpca-nf: templates/qc_report/celltypes_supplemental_report.rmd

  )
}

if (has_consensus) {
  methods_text <- glue::glue(
    "{methods_text}

>     * Consensus cell types obtained by looking for agreement between `SingleR` and `CellAssign` using an ontology-aware approach.

Reviewer: ```suggestion
    * Consensus cell types based on agreement between `SingleR` and `CellAssign` using an ontology-aware approach.
```
